django_admin_index/translations.py

```diff
@@ -0,0 +1,40 @@
+from django.conf import global_settings
```

Comment:
this should be `settings` instead of `global_settings` - projects can decide to support langues that Django does not ship translation catalogs for, and the current implementation does not allow for that.

You'd also want to narrow down these possible translations to the context of supported language of the project - I don't see why you'd enter translations for a language that will never be activated in the UI.

Reply:
Agree, I'm not sure why my thoughts went to support all languages even when they're not available.